include/openssl/trace.h

```diff
@@ -57,8 +57,9 @@ extern "C" {
 # define OSSL_TRACE_CATEGORY_DECODER            15
 # define OSSL_TRACE_CATEGORY_ENCODER            16
 # define OSSL_TRACE_CATEGORY_REF_COUNT          17
+# define OSSL_TRACE_CATEGORY_HTTP               18
```

Comment:
You also need to add an entry to `trace_categories` in trace.c

Reply:
> You also need to add an entry to `trace_categories` in trace.c

Thanks for this hint - I was already wondering how to get "HTTP" recognized by the `openssl` app.
Done, and added a comment both in `trace.h` and `trace.c` that the two lists need to be kept in sync.